packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -126,13 +205,21 @@ const NewNetwork = decorate([
             vlan,
           })
         : isPrivate
-        ? createPrivateNetwork({
-            poolId: pool.id,
-            networkName: name,
-            networkDescription: description,
-            encapsulation: encapsulation,
-            pifId: pif.id,
-          })
+        ? networks.length > 0
```

Comment:
`> 1`?

Reply:
No, `networks` represent only the supplementary pools networks added to the network.
If the user create a pool-wide `network`, networks will be empty.